lib/feature_flagger/configuration.rb

```diff
@@ -0,0 +1,37 @@
+module FeatureFlagger
+  class Configuration
+    attr_reader :storage, :yaml_filepath, :info
+
+    def initialize
+      configure_storage
+      configure_yaml_filepath
+      configure_info
+    end
+
+    def storage=(storage)
+      @storage = storage.nil? ? default_client : storage
```

Comment:
I just did that because of this spec https://github.com/ResultadosDigitais/feature_flagger/pull/17/files#diff-1471e2e6e97c2400b4b079ebf2df4416R25, this test if we pass a nil value for storage, it will keep returning the default value instead assign nil for storage.

If I remove this assign method, the spec fail. :thought_balloon: Any suggestion ?